docs/specifications/tests/Connectivity-TP/connectivity02.md

```diff
@@ -1,39 +1,48 @@
-## CONNECTIVITY02: TCP connectivity
+# CONNECTIVITY02: TCP connectivity
 
-### Test case identifier
+## Test case identifier
 
-**CONNECTIVITY02:** TCP connectivity
+**CONNECTIVITY02**
 
-### Objective
+## Objective
 
-DNS queries are sent using TCP on port 53, as described in section 4.2.2 of
-[RFC 1035](https://tools.ietf.org/html/rfc1035).
+DNS queries are sent using TCP on port 53, as described in
+[RFC 1035#section-4.2.2][RFC 1035], section 4.2.2. As specified in
+[RFC 7766#section-1][RFC 7766], section 1, support of TCP is mandatory.
```

Comment:
These links are messed up. Also, have you considered making the link text "RFC 1035, section 4.2.2"?

Reply:
Link and display text should be the other way around. Will fix.

No, I prefer a shorter display text. This is how I have changed the specifications. E.g. see "objective" in the specification for [DNSSEC10].

[DNSSEC10]: https://github.com/zonemaster/zonemaster/blob/master/docs/specifications/tests/DNSSEC-TP/dnssec10.md#objective